tests/unit/test_workflow_files.py

```diff
@@ -1817,10 +1818,12 @@ def test_check_flow_file(
 
 
 def test_detect_both_flow_and_suite(tmp_path):
-    """Test flow.cylc and suite.rc together in dir raises error."""
+    """Test flow.cylc and suite.rc (as files) together in dir raises error."""
     tmp_path.joinpath(WorkflowFiles.FLOW_FILE).touch()
     tmp_path.joinpath(WorkflowFiles.SUITE_RC).touch()
 
+    forbidden = is_forbidden(tmp_path / WorkflowFiles.FLOW_FILE)
+    assert forbidden is True
     with pytest.raises(WorkflowFilesError) as exc:
         detect_both_flow_and_suite(tmp_path)
         assert str(exc.value) == (
```

Comment:
Just spotted, this `assert` will never be run, because `detect_both_flow_and_suite()` raises and so it will exit out of the `with` block at the point. The `assert` needs to be un-indented